# We enable HEVC and this triggers a ChromeOS related error:
# chrome_key_systems.cc:186:40: error: no member named 'features' in namespace 'chromeos'
# This patch (from Arch User Repository) prevents the build failure.

diff --git a/chrome/renderer/media/chrome_key_systems.cc.orig b/chrome/renderer/media/chrome_key_systems.cc
index 2160697..c9de40c 100644
--- a/chrome/renderer/media/chrome_key_systems.cc.orig
+++ b/chrome/renderer/media/chrome_key_systems.cc
@@ -182,8 +182,12 @@ static SupportedCodecs GetSupportedCodecs(
 #endif  // BUILDFLAG(USE_PROPRIETARY_CODECS)
 #if BUILDFLAG(ENABLE_PLATFORM_HEVC)
       case media::VideoCodec::kCodecHEVC:
+#if BUILDFLAG(IS_CHROMEOS_ASH)
         if (is_secure && base::FeatureList::IsEnabled(
                              chromeos::features::kCdmFactoryDaemon)) {
+#else
+        if (is_secure) {
+#endif
           supported_codecs |= media::EME_CODEC_HEVC_PROFILE_MAIN;
           supported_codecs |= media::EME_CODEC_HEVC_PROFILE_MAIN10;
         }
